xbmc/interfaces/python/PythonSwig.cpp.template

```diff
@@ -916,7 +916,6 @@ namespace PythonBindings
 
    // constants
    PyModule_AddStringConstant(module, "__author__", "Team Kodi <http://kodi.tv>");
-   PyModule_AddStringConstant(module, "__date__", "${new Date().toString()}");
```

Comment:
I would use `CCompileInfo::GetBuildDate`, which can be already set to a specific date via the build system, instead of removing it.

Reply:
Done.